projects/packages/forms/src/dashboard/components/response-actions/index.tsx

```diff
@@ -90,7 +91,7 @@ const ResponseActions = ( {
 	}, [ response, registry, onActionComplete ] );
 
 	const sharedProps = {
-		iconSize: 24,
+		variant: 'tertiary',
```

Comment:
I just [removed](https://github.com/Automattic/jetpack/pull/45845) the variant recently bacause it's causing blue links which I think we'd want to avoid for actions. :-)

Row for comparison:

<img width="234" height="84" alt="Image" src="https://github.com/user-attachments/assets/6f9d7fe6-ef9b-491a-b2a9-cc23c06950e4" />